templates/company/contracts/CompanyTemplate.sol

```diff
@@ -47,18 +47,18 @@ contract CompanyTemplate is BaseTemplate {
         return token;
     }
 
-    function newInstance(string _id, address[] _holders, uint256[] _stakes, uint64[] _voteSettings, uint64 _financePeriod, bool _useAgentAsVault) public {
+    function newInstance(string _id, address[] _holders, uint256[] _stakes, uint64[3] _votingSettings, uint64 _financePeriod, bool _useAgentAsVault) public {
         require(_holders.length > 0, ERROR_EMPTY_HOLDERS);
         require(_holders.length == _stakes.length, ERROR_BAD_HOLDERS_STAKES_LEN);
-        require(_voteSettings.length == 3, ERROR_BAD_VOTE_SETTINGS);
+        require(_votingSettings.length == 3, ERROR_BAD_VOTE_SETTINGS);
         MiniMeToken token = _popTokenCache(msg.sender);
 
         // Create DAO and install apps
         (Kernel dao, ACL acl) = _createDAO();
         Vault agentOrVault = _useAgentAsVault ? _installDefaultAgentApp(dao) : _installVaultApp(dao);
         Finance finance = _installFinanceApp(dao, agentOrVault, _financePeriod == 0 ? DEFAULT_FINANCE_PERIOD : _financePeriod);
         TokenManager tokenManager = _installTokenManagerApp(dao, token, TOKEN_TRANSFERABLE, TOKEN_MAX_PER_ACCOUNT);
-        Voting voting = _installVotingApp(dao, token, _voteSettings[1], _voteSettings[2], _voteSettings[0]);
+        Voting voting = _installVotingApp(dao, token, _votingSettings[0], _votingSettings[1], _votingSettings[2]);
```

Comment:
As mentioned elsewhere, let's use the `Voting.initialize()` parameter order here and in the BaseTemplate to avoid confusion.

Reply:
Note that this change was in favor of that, this is `support, acceptance, duration`